### DISCUSSION TITLE
Autolink Update API

### DISCUSSION THREAD

**Discussion by RyanSiu1995** [community/community#38584] [Apps, API and Webhooks]
Autolink is a really good feature in GitHub.
However, it only provides create, read and delete API.
Could GitHub also provide update API for the community to manage the Autolink easier?

**Comment by ossanna16** [comment]
Hi there @RyanSiu1995 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
